data-plane/python-bindings/srpc/srpc/common.py

```diff
@@ -78,13 +76,15 @@ def service_and_method_to_pyname(
 
 
 def handler_name_to_pyname(
-    name: slim_bindings.PyName, rpc_handler: Rpc
+    name: slim_bindings.PyName,
+    service_name,
+    method_name,
```

Comment:
Question: @Tehsmash , shouldn't we add some type hints here? 😄
(just trolling on the py.typed/mypy thing here)

Same at
data-plane/python-bindings/srpc/srpc/server.py : 48 @ `register_rpc`